docs/11-pages/06-proxies/index.md

```diff
@@ -5,9 +5,9 @@ description: Learn how to set up up a proxy in Nordcraft to serve content from d
 
 # Configure proxies
 
-You can use proxies to map one URL to another URL. Using a proxy hides the path of the destination on the server, so that it appears that the location does not change.
+You can use proxies to map one URL to another URL. Using a proxy hides the path of the destination on the server, so that it appears that the location does not change. A proxy supports all HTTP methods (GET, POST, PUT, DELETE, etc.) and uses the same method, headers and body as the original request when forwarding the request to the destination.
```

Comment:
```suggestion
You can use proxies to map one URL to another URL. Using a proxy hides the path of the destination on the server, so that it appears that the location does not change. A proxy in Nordcraft supports all HTTP methods (GET, POST, PUT, DELETE, etc.) and uses the same method, headers and body as the original request when forwarding the request to the destination.
```